lib/log.test.js

```diff
@@ -0,0 +1,27 @@
+jest.mock('winston')
+const winston = require('winston')
+winston.createLogger = jest.fn().mockImplementation(() => {
+    return {
+        error: () => {},
+        warn: () => {},
+        info: () => {}
+    }
+})
+winston.format = {
+    combine: () => {},
+    timestamp: () => {},
+    label: () => {},
+    colorize: () => {},
+    printf: () => {}
+}
+winston.transports.Console = function(){}
+
+describe('Testing log.js', () => {
+    beforeEach(() => {
+        jest.clearAllMocks()
+    })
+    it('should call winston.createLogger when log is required', () => {
+        const logger = require('./log')(__filename)
+        expect(winston.createLogger).toHaveBeenCalled()
+    })
```

Comment:
`createLogger` is called only once when `logger` is defined in `./log` so I changed this part to this from when there were 3 tests that are for `log.error`, `log.warn`, `log.info`